chain/westend-local/default.go

```diff
@@ -5,18 +5,19 @@ package westendlocal
 
 import (
 	cfg "github.com/ChainSafe/gossamer/config"
+	"github.com/adrg/xdg"
 )
 
-const (
+var (
 	// defaultChainSpec is the default chain spec for the westend local node
 	defaultChainSpec = "./chain/westend-local/westend-local-spec-raw.json"
 
-	// DefaultBasePathAlice is the default basepath for the westend local alice node
-	DefaultBasePathAlice = "~/.gossamer/westend-local-alice"
-	// DefaultBasePathBob is the default basepath for the westend local bob node
-	DefaultBasePathBob = "~/.gossamer/westend-local-bob"
-	// DefaultBasePathCharlie is the default basepath for the westend local charlie node
-	DefaultBasePathCharlie = "~/.gossamer/westend-local-charlie"
+	// defaultBasePathAlice is the default basepath for the westend local alice node
+	defaultBasePathAlice = xdg.DataHome + "/gossamer/westend-dev/alice"
+	// defaultBasePathBob is the default basepath for the westend local bob node
+	defaultBasePathBob = xdg.DataHome + "/gossamer/westend-dev/bob"
+	// defaultBasePathCharlie is the default basepath for the westend local charlie node
+	defaultBasePathCharlie = xdg.DataHome + "/gossamer/westend-dev/charlie"
```

Comment:
Why does path in westend local have `dev` in it? Is this deliberate?

Reply:
No, another oversight 🤦🏼 
Thanks